performance.rst

```diff
@@ -6,43 +6,66 @@ Performance
 
 Symfony is fast, right out of the box. Of course, if you really need speed,
 there are many ways that you can make Symfony even faster. In this chapter,
-you'll explore many of the most common and powerful ways to make your Symfony
-application even faster.
+you'll explore some of the ways to make your Symfony application even faster.
 
 .. index::
    single: Performance; Byte code cache
 
-Use a Byte Code Cache (e.g. APC)
---------------------------------
+Use a Byte Code Cache (e.g. OPcache)
+------------------------------------
+
+The first thing that you should do to improve your performance is to use a
+"byte code cache". These caches store the compiled PHP files to avoid having
+to recompile them for every request.
 
-One of the best (and easiest) things that you should do to improve your performance
-is to use a "byte code cache". The idea of a byte code cache is to remove
-the need to constantly recompile the PHP source code. There are a number of
-`byte code caches`_ available, some of which are open source. As of PHP 5.5,
-PHP comes with `OPcache`_ built-in. For older versions, the most widely used
-byte code cache is probably `APC`_
+There are a number of `byte code caches`_ available, some of which are open
+source. As of PHP 5.5, PHP comes with `OPcache`_ built-in. For older versions,
+the most widely used byte code cache is `APC`_.
 
 Using a byte code cache really has no downside, and Symfony has been architected
 to perform really well in this type of environment.
 
 Further Optimizations
 ~~~~~~~~~~~~~~~~~~~~~
 
-Byte code caches usually monitor the source files for changes. This ensures
-that if the source of a file changes, the byte code is recompiled automatically.
-This is really convenient, but obviously adds overhead.
+Most byte code caches monitor the source files for changes. This ensures that if
+the source of a file changes, the byte code is recompiled automatically.
+This is really convenient, but it adds overhead.
 
 For this reason, some byte code caches offer an option to disable these checks.
-Obviously, when disabling these checks, it will be up to the server admin
-to ensure that the cache is cleared whenever any source files change. Otherwise,
-the updates you've made won't be seen.
+For example, to disable these checks in APC, simply add ``apc.stat=0`` to your
+``php.ini`` configuration.
+
+When disabling these checks, it will be up to the server administrators to
+ensure that the cache is cleared whenever any source files change. Otherwise,
+the updates you've made in the application won't be seen.
 
-For example, to disable these checks in APC, simply add ``apc.stat=0`` to
-your ``php.ini`` configuration.
+For the same reasons, the byte code cache must also be cleared when deploying
+the application (for example by calling ``apc_clear_cache()`` PHP function when
+using APC and ``opcache_reset()`` when using OPCache).
```

Comment:
We should also note that the CLI and the web process do not share the same Opache (which will otherwise probably trick you when you clear the Opcache through your deploy script only for the CLI environment).